portfolio/src/main/webapp/script.js

```diff
@@ -49,3 +49,14 @@ function addRandomDestination() {
 
   travelContainer.innerText = placeToVisit;
 }
+
+
+/** 
+ * adds response from servlet
+ */
+
+function getGreeting() {
```

Comment:
nit: not sure if the instructions say to name your method this way (maybe this is normal in-browser JS style?), but it bothers me a little bit that this says `getGreeting` when it doesn't return anything. This could be something more like `setGreetingFromServerData` or something that indicates that its actual purpose is mutating the `greeting-container` div.